test/list.spec.js

```diff
@@ -0,0 +1,37 @@
+import { mount } from '@vue/test-utils';
+import list from '../src/components/list.vue';
+import listitem from '../src/components/listitem.vue'
+
+describe('list.vue', () => {
+  const items = [{text: '1'}, {text: '2'}, {text:'3'}]
+  const wrapper = mount(list, {
+    propsData: { items: items }
+  });
+  it('renders a list', () =>{
+    expect(wrapper.contains('div')).toBe(true);
+    expect(wrapper.findAll('p')).toHaveLength(items.length)
```

Comment:
```suggestion
    expect(wrapper.findAll(listitem)).toHaveLength(items.length)
```
You can even do that :point_up:

Reply:
And also use hard-coded values in the assertion for safety:
```suggestion
    expect(wrapper.findAll('p')).toHaveLength(3)
```